File: mina/JAVA/baekjoonjava/src/B10807.java
import java.util.Scanner;
class B10807 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int num[] = new int[N];
        for(int i=0; i<N; i++){
            num[i] = sc.nextInt();
        }
        int v = sc.nextInt();
        int count = 0;
        for(int i=0; i<N; i++){
            if(v == num[i]){
                count +=1;
            }
        }
        System.out.println(count);
    }

}

File: mina/C++/baekjoonC++/baekjoonC++/B10807.cpp
#include <iostream>
using namespace std;
int main() {

	int N = 0;
	cin >> N;
	int* num = new int[N]; //포인터배열
	for (int i = 0; i < N; i++) {
		cin >> num[i];
	}
	int v = 0;
	cin >> v;
	int count = 0;
	for (int i = 0; i < N; i++) {
		if (num[i] == v)
			count += 1;
	}
	cout << count << endl;
	return 0;
};